packages/studio-base/src/dataSources/FoxgloveWebSocketDataSourceFactory.tsx

```diff
@@ -14,6 +14,8 @@ export default class FoxgloveWebSocketDataSourceFactory implements IDataSourceFa
   type: IDataSourceFactory["type"] = "connection";
   displayName = "Foxglove WebSocket";
   iconName: IDataSourceFactory["iconName"] = "Flow";
+  docsIntro = "Connect live to your non-ROS data via an encoding-agnostic WebSockets connection.";
```

Comment:
```suggestion
  docsIntro = "Connect live to your non-ROS data via an encoding-agnostic WebSocket connection.";
```

Not sure if we should say "non-ROS" here. If someone doesn't use ROS, then the fact that this is non-ROS might not be so relevant to them 😀   Maybe "custom data"?

I think it might be good to mention that in order to use this data source, you usually have to write a custom server.